.github/workflows/moduletest-file-reset-build-release.yaml

```diff
@@ -0,0 +1,92 @@
+# Build the container image for the DPL shell on any modifications to the
+# workflow or the shell project.
+# The build is then tested, and if we where triggered by a tag, a tagged
+# version of the image is pushed to a public registry.
+
+on:
+  pull_request:
+    paths:
+      - "/infrastructure/images/moduletest-file-reset-jobs/**"
+      - ".github/workflows/moduletest-file-reset-build-release.yaml"
+    tags:
+      - "mfr-*-rc"
```

Comment:
Could you document this tag wildcard usage? Why does it look like this?

Reply:
done